### PR TITLE
BZ-2084484 First commit for updates to MetalLB

### PR DESCRIPTION
**Pull request by kquinn1204**
[BZ-2084484]: Updates to MetalLB documentation

Version(s): 4.10

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2084484

Link to docs preview:
http://file.emea.redhat.com/kquinn/BZ-2084484/networking/metallb/metallb-operator-install.html

http://file.emea.redhat.com/kquinn/BZ-2084484/networking/metallb/about-metallb.html

http://file.emea.redhat.com/kquinn/BZ-2084484/networking/metallb/about-metallb.html#nw-metallb-operator-custom-resources_about-metallb-and-metallb-operator

### CODE REVIEW

**Comment by evgenLevin** [comment]
LGTM

File: modules/nw-metallb-installing-operator-cli.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-metallb-installing-operator-cli_{context}"]
-= Installing from OperatorHub using the CLI
+= Installing the MetalLB Operator from the OperatorHub using the CLI
```
Comment (fedepaol) [inline modules/nw-metallb-installing-operator-cli.adoc:7]: I think from OperatorHub is fine

File: modules/nw-metallb-installing-operator-cli.adoc [hunk 2]
```diff
@@ -101,7 +99,7 @@ metadata:
   name: metallb-operator-sub
   namespace: metallb-system
 spec:
-  channel: "${OC_VERSION}"
+  channel: "stable"
```
Comment (fedepaol) [inline modules/nw-metallb-installing-operator-cli.adoc:102]: is it always stable? Shouldn't be 4.10 something?

File: networking/metallb/about-metallb.adoc [hunk 3]
```diff
@@ -10,7 +10,7 @@ As a cluster administrator, you can add the MetalLB Operator to your cluster so
 The external IP address is added to the host network for your cluster.
 
 You can configure MetalLB so that the IP address is advertised with layer 2 protocols.
-With layer 2, MetalLB provides a fault-tolerant external IP address.
+With layer 2, MetalLB provides a fault-tolerant external IP address.
```
Comment (fedepaol) [inline networking/metallb/about-metallb.adoc:13]: nit: extra white space

File: modules/nw-metallb-installing-operator-cli.adoc (hunk 2 above)
Comment (kquinn1204) [reply modules/nw-metallb-installing-operator-cli.adoc:102]: I checked and this command `oc get packagemanifest metallb-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel} ` returned `stable`. I know around 4.10 timeframe some Operators were moved to stable channel. Maybe @evgenLevin can verify, but I recall making updates to other Operators from `OC_VERSION` to stable in 4.10 release

Comment (kquinn1204) [reply modules/nw-metallb-installing-operator-cli.adoc:102]: I tried 4.11 and MetalLB Operator failed to install. It only seems to install with channel set to `stable`.

Comment (fedepaol) [reply modules/nw-metallb-installing-operator-cli.adoc:102]: That's because 4.11 is not out yet

Comment (kquinn1204) [reply modules/nw-metallb-installing-operator-cli.adoc:102]: OK going to try with 4.10 I maybe mistakenly thought channel was updated to stable for this Operator from 4.10 >

Comment (kquinn1204) [reply modules/nw-metallb-installing-operator-cli.adoc]: reverted to original content as you state works with 4.10

File: modules/nw-metallb-software-components.adoc [hunk 4]
```diff
@@ -28,9 +28,7 @@ The `speaker` uses Address Resolution Protocol (ARP) to announce IPv4 addresses
 
 For BGP mode, after the `controller` allocates an IP address for the service, each `speaker` pod advertises the load balancer IP address with its BGP peers.
 You can configure which nodes start BGP sessions with BGP peers.
-+
-Refer to the concepts for layer 2 mode and BGP mode for more information.
-+
+
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply modules/nw-metallb-software-components.adoc:31]: @kquinn1204 While this has been resolved, I am not seeing the update. Do you need to push another commit?

File: modules/nw-metallb-operator-custom-resources.adoc [hunk 5]
```diff
@@ -34,7 +34,7 @@ BFD provides faster path failure detection than BGP alone provides.
 After you add the `MetalLB` custom resource to the cluster and the Operator deploys MetalLB, the MetalLB software components, `controller` and `speaker`, begin running.
 
 The Operator includes validating webhooks for the `AddressPool` and `BGPPeer` custom resources.
-The webhook for the address pool custom resource performs the following checks: that the address pool name is unique.
+The webhook for the address pool custom resource performs the following checks:
```
Comment (mjpytlak) [inline modules/nw-metallb-operator-custom-resources.adoc:37]: @kquinn1204 I did miss this the first time through. Apologies. While you are here, please add the `:_content-type: <type>`

**Review by kelbrown20 — COMMENTED** [review]
LGTM!

**Pull request merged**